backend/classes/production/sefilm/seeds/locations.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
This file should not be committed. I thought I had removed the folder from .gitignore...